### ISSUE TITLE
Blank cells in .xlsx would return NaN in the markdown.

### ISSUE THREAD

**Issue by RealNath** [microsoft/markitdown#183]
I just thought Python's default `NaN` doesn't look visually good on the markdown results. There should be a way/an option for it to output an empty cell instead.

**Comment by l-lumin** [comment]
related to
- #52

**Comment by johann-petrak** [comment]
Yes, it should be trivial to make sure that Markdown table cells for empy sheet cells are empy instead of containing "NaN" .

This is extremely annoying

**Comment by fn5** [comment]
Following up on this item.

In excel:

<img width="1465" height="58" alt="Image" src="https://github.com/user-attachments/assets/44156cb9-cf5d-4365-ae75-50f460296950" />

Using markitdown:

`| • The R-X points that define the polygon has been identified. | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN |
| NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN |
| For each harmonic, the attached spreadsheet contains: | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN |`

**Comment by dan-i-am** [comment]
On the other hand, if the markdown output is handed to certain LLMs, having a NaN value can sometimes help it track column ordinals better and reduce mismatch-related hallucinations.  It also allows the LLM to infer a "numeric type" since NaN implies it was a numeric cell with a "null/blank" value in it.

My advice is that excel conversion should have a formatting config that allows NaN to optionally be replaced by something else (empty string or some other string) vs. making a blanket assumption to always remove NaN as not useful.  There are other formatting things missing from the export that would be optionally useful (such as date and currency formats).

**Comment by WolffM** [comment]
hey, i attempted a fix for this. Let me know if it looks right: https://github.com/microsoft/markitdown/pull/1619
